.gitignore

```diff
@@ -1,2 +1,5 @@
 /vendor/
 .phpcs-cache
+.php_cs.cache
+.php-cs-fixer.cache
+/.idea/
```

Comment:
I usually prefer to keep not project related things, as IDE data, out of the `.gitignore` (instead I ignore them globally on my machine)

Reply:
could you please add an empty line at the of the file?

Reply:
> I usually prefer to keep not project related things, as IDE data, out of the `.gitignore` (instead I ignore them globally on my machine)

I understand... but someone else might not do that :)

Going to remove that line and add the empty line.